ChangeLog

```diff
@@ -10,6 +10,10 @@ Release date: TBA
   Put new features here and also in 'doc/whatsnew/2.14.rst'
 
 
+* Fix false positive for ``unnecessary-ellipsis`` when using an ellipsis as a default argument.
```

Comment:
Could you move this to `2.13.3` and move it to the `2.13.rst` file as well? I have one fix that needs a new patch release anyway, so we might as well release this with it!

Reply:
Moved it over now 😄